apps/user/views.py

```diff
@@ -31,11 +31,32 @@ class UserCreate(generics.CreateAPIView):
     permission_classes = (permissions.AllowAny,)
 
 
-class UserList(generics.ListAPIView):
+class UserList(LoggingMixin, generics.ListAPIView):
+    # optionally specify what methods to log
+    # logging_methods = ['GET','POST']
     queryset = get_user_model().objects.all()
     serializer_class = UserSerializer
     permission_classes = (permissions.AllowAny,)
 
+    '''
+    optional mask sensitive data. By default drf-tracking is hiding the
+    values of the fields {'api', 'token', 'key', 'secret', 'password', 'signature'}.
+    '''
+    sensitive_fields = {'password'}
```

Comment:
Does setting this to just password override the other defaults (ie no longer hides 'api', 'token', 'key', 'secret', 'signature')?

Reply:
I didn't test that specifically, but I wouldn't think so.